documentation/FAQ.md

```diff
@@ -0,0 +1,39 @@
+# FAQ in Swagger PR Review
+
+This page is intended to answer questions frequently asked during Azure Swagger PR review.
+
+1. [I am new to Swagger/OpenAPI, How should I start?](#Onboard)
+2. [How to fix validation failure?](#validation)
+3. [How to generate SDK from Swagger?](#SDK)
+4. [How to generate document](#doc)
+5. [If need further help, who can we contact?](#contact)
+
+## How to onboard PR review Process?<a name="Onboard"></a>
+If you are new to Swagger/OpenAPI, you can refer to this [document](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/80/Getting-started-with-OpenAPI-specifications)
+
+If you are new to Swagger PR review process, you can refer to this [flowchart](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review)
+
+You can refer to this [document](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/49/Request-Access-to-Azure-SDK-repos) to get read permission to submit PR.
+
+## How to fix validation failure?<a name="validation"></a>
+| Validation | Description | How to fix |
+| --- | --- | --- |
+| [Modal Validation](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) | Validates models defined for body parameters and responses, and matches them against the expected input and output of an operation. | [Here](https://aka.ms/ci-fix#model-validation) |
```

Comment:
For the description of model validation, consider reference https://github.com/Azure/oav/blob/master/README.md#what-does-the-tool-do-what-issues-does-the-tool-catch

Reply:
Thanks, changed.